_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -148,6 +181,23 @@ Label is included in accessibility tree, but not visible, and does not describe
 <input aria-labelledby="label_fname" type="text" name="fname"/>
 ```
 
+#### Failed example 6
+
+Accessible name created through "aria-label" does not describes the purpose of the associated element.
+
+```html
+<input aria-label="Menu" id="fname" type="text" name="fname"/>
+```
+
+#### Failed example 7
+
+The `label` in itself does not describe the purpose of the form field, and while the context can be relied upon to differentiate form fields from each other, it is not sufficient to let rely on the context for describing the entire purpose of the form field.
+
+```html
+<h2>Name</h2>
+<label>Fill in: <input id="name" type="text" name="name"/></label>
+````
+
 ### Inapplicable
```

Comment:
In the code of the first 3 examples correct 
`style:`
to
`style=`

In inapplicable example 3 I suggest removing the `style` attribute because it detracts from the inapplicability reason that is being highlighted.